.copr/Makefile

```diff
@@ -1,4 +1,5 @@
 srpm:
+	dnf5 -y install 'dnf5-command(builddep)'
```

Comment:
I don't think we need to explicitly type `dnf5` here, but it doesn't hurt either.

Reply:
Right. It's wired indeed. I'll send another PR to fix this.